plugins/qtosgrave/qtosgviewer.cpp

```diff
@@ -1029,98 +1033,9 @@ void QtOSGViewer::_UpdateCameraTransform(float fTimeElapsed)
 {
```

Comment:
What do you mean it does not take the parameter anymore?

Reply:
`fTimeElapsed` is not needed anymore in `_UpdateCameraTransform`

1. It should be removed from arguments
2. It used to have this to sync with environment update for close-in animation when tracking anything. Is it ok to just use `QTime`?

Reply:
I see. _UpdateEnvironment(float fTimeElapsed) call UpdateCameraTransform.

QTime has enough precision for FPS measure, I can change to QTime.

Reply:
Wait, the only function that used delta time was _UpdateCameraTransform, but it doesnt use anymore.. Dont need to use QTime, do we?

Reply:
And actually I think maybe _UpdateCameraTransform can now be called _Update only, since it does other things now.. dont need to update camera transform, but other general things

Reply:
I meant in your camera manipulator you are using a `QTime` to measure elapsed time, is it Ok not to use this `fTimeElapsed`? in order to in sync with environment update.

Reply:
Its ok. Its for the osg animation only. OSG also measure real frame delta time for everything so it can be framerate indepent. So in the end I could even use some other OSG function to measure, but I think using QTime is easier and cleaner imho.